frontend/packages/core/src/Input/tests/toggle-button-group.test.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from "react";
+import { ToggleButton } from "@material-ui/lab";
+import { shallow } from "enzyme";
+
+import ToggleButtonGroup from "../toggle-button-group";
+
+// Note interaction tests will happen when there is a parent component that
+// can maintain state and select a default
+describe("ToggleButtonGroup", () => {
+  let multipleFalseWrapper;
+  let multipleTrueWrapper;
+  beforeEach(() => {
+    multipleFalseWrapper = shallow(
```

Comment:
nit: It might be nicer to turn these wrappers into a function that accepts props on the ButtonGroup and returns the shallow wrapped component. This way we can expand the usage of this easily and we don't need two separate pieces to represent multi/non-multi.